lib/modules/saveComments.js

```diff
@@ -167,122 +159,118 @@ addModule('saveComments', function(module, moduleID) {
-		savedCommentsContent.appendChild(saveRESTips);
+		RESTemplates.load('saveComments', function(template) {
+			var output = template.html({
+				moduleDescription: module.description,
+				keyNavHash: modules['settingsNavigation'].makeUrlHash('keyboardNav', 'savePost'),
+				savePostKey: modules['keyboardNav'].getNiceKeyCode('savePost'),
+				saveCommentKey: modules['keyboardNav'].getNiceKeyCode('saveComment'),
+				saveRESKey: modules['keyboardNav'].getNiceKeyCode('saveRES')
+			});
+			$('body > .content').append(output);
+		});
 
-		for (var i in storedComments) {
-			if ((i !== 'RESPro_add') && (i !== 'RESPro_delete')) {
-				var clearLeft = document.createElement('div');
-				clearLeft.setAttribute('class', 'clearleft');
-				var thisComment = document.createElement('div');
-				thisComment.classList.add('savedComment');
-				thisComment.classList.add('entry');
-				// this is all saved locally, but just for safety, we'll clean out any script tags and whatnot...
-				// we'll also remove iframe / video tags because they might autoplay.
-				// TODO: save comments using markdown instead of HTML in the future. It's cleaner/safer/better.
-				var cleanHTML = '<div class="savedCommentHeader">Comment by user: ' + escapeHTML(storedComments[i].username) + ' saved on ' + escapeHTML(storedComments[i].timeSaved) + '</div>';
-				cleanHTML += '<div class="savedCommentBody md">' + storedComments[i].comment.replace(/<(script|iframe|video)(.|\s)*?\/(script|iframe|video)>/g, '') + '</div>';
-				cleanHTML += '<div class="savedCommentFooter"><ul class="flat-list buttons"><li><a class="unsaveComment" href="javascript:void 0">unsave-RES</a></li><li><a href="' + escapeHTML(storedComments[i].href) + '">view original</a></li></ul></div>';
-				$(thisComment).html(cleanHTML);
-				var unsaveLink = thisComment.querySelector('.unsaveComment');
-				$(unsaveLink)
-					.data('unsaveID', i)
-					.data('unsaveLink', storedComments[i].href);
-				unsaveLink.addEventListener('click', function(e) {
-					e.preventDefault();
-					unsaveComment($(this).data('unsaveID'));
-				}, true);
-				savedCommentsContent.appendChild(thisComment);
-				savedCommentsContent.appendChild(clearLeft);
-			}
+		$wrapper = $('#res-saveComments');
+
+		if (Object.getOwnPropertyNames(storedComments).length) {
+			$wrapper.find('.res-module-description').hide();
+		}
+
+		if (!modules['keyboardNav'].isEnabled()) {
+			$wrapper.find('.res-module-tip').hide();
```

Comment:
This can be done in the template with `{{#keyboardNavEnabled}}...{{/keyboardNavEnabled}}`.

That is: if `foo` isn't an array, then `{{#foo}}bar{{/foo}}{{^foo}}baz{{/foo}}` acts like `foo ? 'bar' : 'baz'`.